lib/sensors/accelerometer.cpp

```diff
@@ -13,6 +42,44 @@ Accelerometer::~Accelerometer()
 {
 }
 
+std::optional<core::Result> Accelerometer::isVaueReady()
```

Comment:
optional result is not a great type imo, but i understand there might not be a better way to do this

Reply:
Also `isValueReady`

Reply:
yeah not sure how to do this better atm. in my head this is running on a ros node where isValueReady is called before read